recipes/python-symengine/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export CC=clang
+    export CXX=clang++
+
+    export MACOSX_VERSION_MIN=10.9
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+    export LINKFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11 -L${LIBRARY_PATH}"
+
+    # This enables Xcode support for C++11, that is needed for SymEngine.
+    export MACOSX_DEPLOYMENT_TARGET=10.9
+fi
```

Comment:
No, it sets it to 10.7, which is what we should be doing. It will still be compatible with 10.9. I use 10.9 regularly with binaries built here.